packages/boa/lib/index.js

```diff
@@ -10,6 +10,15 @@ const DelegatorLoader = require('./delegator-loader');
 // internal symbols
 const IterIdxForSeqSymbol = Symbol('The iteration index for sequence');
 
+// read the conda path from the .CONDA_INSTALL_DIR
+// eslint-disable-next-line no-sync
+const condaPath = fs.readFileSync(path.join(__dirname, '../.CONDA_INSTALL_DIR'), 'utf8');
```

Comment:
```suggestion
const condaPath = fs.readFileSync(path.join(__dirname, '..', '.CONDA_INSTALL_DIR'), 'utf8');
```
for more cross-platform adaptability.

Reply:
And could we use async function？

Reply:
unfortunately, we have to use this function here, because the top-level await is not supported.

Reply:
> adaptability

The `path.join` resolved the cross-platform issue in this way, you could try:

```
> path.win32.join('foobar', '123/test')
'foobar\\123\\test'
```